src/ticketmap/index.js

```diff
@@ -297,18 +297,26 @@ export default class TicketMap extends Component<*, State> {
    * Helpers
    */
 
-  getAvailableTicketGroups = (availableTicketGroups = []) =>
-    availableTicketGroups.reduce((memo, { tevo_section_name, retail_price: price }) => {
+  getAvailableTicketGroups = (availableTicketGroups = []) => {
+    const unknownSectionNames = []
+
+    const result = availableTicketGroups.reduce((memo, { tevo_section_name, retail_price: price }) => {
       const section = tevo_section_name.toLowerCase()
       const sectionZoneMeta = this.state.sectionZoneMapping[section]
       if (sectionZoneMeta) {
         memo.push({ section, price, zone: sectionZoneMeta.zone })
-        console.log(`Section ${section} was successfully mapped!`)
       } else {
-        console.warn(`Section ${section} not found. Please verify it exists in the venue manifest`)
+        unknownSectionNames.push(section)
       }
       return memo
-    }, []);
+    }, [])
+
+    if (unknownSectionNames.length > 0) {
+      console.error('Unknown section names found in ticket groups: %o', unknownSectionNames)
```

Comment:
Not sure if this warrants an error. It doesn't prevent the application from running properly, it's more of an expected TODO for our team.

Reply:
I hear you, I can see this as a warning instead of an error